files/justfiles/toggles.just

```diff
@@ -222,26 +222,17 @@ toggle-bash-environment-lockdown:
       echo "Capitalization matters when you type \"YES I UNDERSTAND\""
     fi
 
-# Toggle unconfined domain userns creation
-toggle-unconfined-domain-userns-creation:
+# Toggle either unconfined or container domain userns creation
+toggle-userns-creation MODULE:
     #! /bin/run0 /bin/bash
-    MODULE_NAME="harden_userns"
-
-    if semodule -l | grep -q "$MODULE_NAME"; then
-        echo "Module $MODULE_NAME is currently enabled. Disabling it..."
-        semodule --disable="$MODULE_NAME"
-        echo "Module $MODULE_NAME disabled."
+    if [ {{ MODULE }} == container ] ; then
```

Comment:
tbh I prefer these separate. otherwise users will have to know the module names, which is very inconvenient

Reply:
If they provide an unsupported arg, a message will advise them: `echo "Invalid input. Valid options are 'container' or 'unconfined'."`

Reply:
Right, but then they still need to know the options. if anything this would need to be a selector

but tbh I thinkit's reasonable as-is, since it lets users see both options from `ujust --choose`

Reply:
Fair enough. Though I'll recommend revisiting this if the number of selinux module toggles grows, or if it becomes deemed that there are too many ujust commands in total.

Reply:
The number of modules will grow, the number of toggles won't need to though. For example look at the chromium and flatpak modules we added, there are no toggles for them because there's no reason to disable them (as doing so would break them)